ddl-tpcds/bin/add_constraints.sql

```diff
@@ -0,0 +1,159 @@
+-- set hivevar:DB=tpcds_bin_orc_10000
+
+alter table customer_address add constraint ${DB}_pk_ca primary key (ca_address_sk) disable novalidate rely;
```

Comment:
Moving the constraints to DDL means we won't support older Hive versions. For example, the NOT NULL constraint is only available since 3.x. Do you think we should only support 3.x and above?

Reply:
OK got it.